db/upgrade_migrations/20160723012349_upgrade_v062_to_v063.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class UpgradeV062ToV063 < ActiveRecord::Migration[5.0]
+  def up
+    i = 1
+    Thredded::MessageboardGroup.order(created_at: :desc).each do |group|
+      Rails.logger.info group.id
+      next if group.valid?
+
+      group.name = "#{group.name}-#{i}"
+      group.save!
+      i += 1
+    end
```

Comment:
The sql could break too? Not sure if there is a way, we could also just make people fix their own dupes since there may need to be human decisions involved.